spec/factories.rb

```diff
@@ -61,19 +61,19 @@ def r_str
   end
 
   factory :like do
-    association :author, :factory => :person
-    association :target, :factory => :status_message
+    association :author, factory: :person
+    association :target, factory: :status_message
   end
 
   factory :user do
-    getting_started false
-    sequence(:username) { |n| "bob#{n}#{r_str}" }
-    sequence(:email) { |n| "bob#{n}#{r_str}@pivotallabs.com" }
-    password "bluepin7"
-    password_confirmation { |u| u.password }
-    serialized_private_key  OpenSSL::PKey::RSA.generate(1024).export
+    getting_started { false }
+    sequence(:username) {|n| "bob#{n}#{r_str}" }
+    sequence(:email) {|n| "bob#{n}#{r_str}@pivotallabs.com" }
+    password { "bluepin7" }
+    password_confirmation {|u| u.password }
```

Comment:
Style/SymbolProc: Pass `&:password` as an argument to `password_confirmation` instead of a block.